originals/g.yaml

```
@@ -283,7 +283,7 @@
 
 - name: Gorillas
   names:
-  - Gorilla.Bas
+  - Gorillas.Bas
```

```suggestion
  - GORILLA.BAS
```
the name comes from the filename which appeared as gorilla.bas